beacon-chain/rpc/validator/server.go

```diff
@@ -148,53 +191,10 @@ func (vs *Server) ValidatorPerformance(
 		AverageActiveValidatorBalance: avgBalance,
 		MissingValidators:             missingValidators,
 		TotalValidators:               uint64(len(headState.Validators)),
-		TotalActiveValidators:         uint64(activeCount),
-	}, nil
-}
-
-// ExitedValidators queries validator statuses for a give list of validators
-// and returns a filtered list of validator keys that are exited.
-func (vs *Server) ExitedValidators(
-	ctx context.Context,
-	req *pb.ExitedValidatorsRequest) (*pb.ExitedValidatorsResponse, error) {
-
-	_, statuses, err := vs.multipleValidatorStatus(ctx, req.PublicKeys)
-	if err != nil {
-		return nil, status.Errorf(codes.Internal, "Could not retrieve validator statuses: %v", err)
-	}
-
-	exitedKeys := make([][]byte, 0)
-	for _, st := range statuses {
-		s := st.Status.Status
-		if s == pb.ValidatorStatus_EXITED ||
-			s == pb.ValidatorStatus_EXITED_SLASHED ||
-			s == pb.ValidatorStatus_INITIATED_EXIT {
-			exitedKeys = append(exitedKeys, st.PublicKey)
-		}
-	}
-
-	resp := &pb.ExitedValidatorsResponse{
-		PublicKeys: exitedKeys,
-	}
-
-	return resp, nil
-}
-
-// DomainData fetches the current domain version information from the beacon state.
-func (vs *Server) DomainData(ctx context.Context, request *pb.DomainRequest) (*pb.DomainResponse, error) {
-	fork := vs.ForkFetcher.CurrentFork()
-	dv := helpers.Domain(fork, request.Epoch, request.Domain)
-	return &pb.DomainResponse{
-		SignatureDomain: dv,
+		TotalActiveValidators:         activeCount,
 	}, nil
 }
 
-// CanonicalHead of the current beacon chain. This method is requested on-demand
-// by a validator when it is their time to propose or attest.
-func (vs *Server) CanonicalHead(ctx context.Context, req *ptypes.Empty) (*ethpb.BeaconBlock, error) {
```

Comment:
Is this supposed to be removed also?

Reply:
Not removed, just moved further up in the file

Reply:
Ah I see now